LayoutTests/editing/selection/deleteFromDocument-shadow-tree-crash.html

```diff
@@ -10,7 +10,10 @@
  function runTest() {
     document.getElementById('input_0').disabled = true;
     document.getElementById('input_0').setRangeText("abc");
-    window.getSelection().extend(document.getElementById('input_0'), 0);
+    try {
+        window.getSelection().extend(document.getElementById('input_0'), 0);
```

Comment:
I don’t understand. If this will throw an exception, why call it? Does it have side effects when it throws the exception?

Also we should remove the unneeded `window.`.